app/models/order.rb

```diff
@@ -0,0 +1,6 @@
+class Order < ActiveRecord::Base
+  has_one :capability
```

Comment:
`Capability` belongs to `contract`

and 

`Order` has_one `capability`
Eh.... I don't understand their part
should this line be inside `Contract`?